packages/libs/error-stack/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "error-stack"
-version = "0.1.1"
+version = "0.2.0"
```

The `publish` job works as expected 🎉 
```suggestion
version = "0.1.1"
```